.trivyignore

```diff
@@ -10,6 +10,3 @@ CVE-2024-27304
 CVE-2023-45288
 CVE-2024-24786
 CVE-2024-24791
-
-#telegraf CRITICAL
```

Comment:
is this gone with adx code removal?

Reply:
this is gone because telegraf has been rebuilt to fix this vuln in the latest mariner which we are consuming